packages/sources/wootrade/schemas/env.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/wootrade-adapter.json",
+  "title": "@chainlink/wootrade-adapter env var schema",
+  "required": ["API_KEY"],
```

Comment:
I think we should just remove it as required here, since the (HTTP) adapter can work without it.

Reply:
removed